include/solvers/heat_transfer.h

```diff
@@ -31,11 +31,15 @@
 #include <deal.II/distributed/tria_base.h>
 
 #include <deal.II/fe/fe_q.h>
+#include <deal.II/fe/mapping_fe.h>
+#include <deal.II/fe/mapping_q.h>
 
 #include <deal.II/lac/trilinos_sparse_matrix.h>
 #include <deal.II/lac/trilinos_vector.h>
 
 #include <core/simulation_control.h>
+#include <deal.II/simplex/fe_lib.h>
+#include <deal.II/simplex/quadrature_lib.h>
```

Comment:
I guess we will have to update this according to the new changes on deal, right?
```suggestion
#include <deal.II/fe/fe_simplex_p.h>
#include <deal.II/base/quadrature_lib.h>
```